sdk/agrifood/agrifood-farming-rest/karma.conf.js

```diff
@@ -52,14 +52,13 @@ module.exports = function (config) {
     },
 
     envPreprocessor: [
-      "RECORDINGS_RELATIVE_PATH",
       "TEST_MODE",
-      "FARMBEATS_ENDPOINT",
```

Comment:
is this expected?

Reply:
yes, this is current template:
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/agrifood/agrifood-farming-rest/karma.conf.js#L55-L62